Code/Editor/ViewportManipulatorController.cpp

```diff
@@ -131,6 +151,12 @@ namespace SandboxEditor
                     }
                     eventType = MouseEvent::Up;
                 }
+                if (SandboxEditor::ManipulatorMouseWrap())
```

Comment:
question: Should this also be `if (SandboxEditor::ManipulatorMouseWrap() && event.m_priority == ManipulatorPriority)`? I think it should be symmetrical to the other condition, is there any reason why that's not also here?